product_docs/docs/pgd/5/consistency/conflicts.mdx

```diff
@@ -210,8 +210,15 @@ If the deleted row is still detectable (the deleted row wasn't removed by `VACUU
 
 The database can clean up the deleted row by the time the `UPDATE` is received in case the local node is lagging behind in replication. In this case, PGD can't differentiate between `UPDATE`/`DELETE` conflicts and [INSERT/UPDATE conflicts](#insertupdate-conflicts). It generates the `update_missing` conflict.
 
-Another type of conflicting `DELETE` and `UPDATE` is a `DELETE` that comes after the row was updated locally. In this situation, the outcome depends on the type of conflict detection used. When using the
-default, [origin conflict detection](#origin-conflict-detection), no conflict is detected, leading to the `DELETE` being applied and the row removed. If you enable [row version conflict detection](#row-version-conflict-detection), a `delete_recently_updated` conflict is generated. The default resolution for this conflict type is to apply the `DELETE` and remove the row. However, you can configure this or a conflict trigger can handled it.
+Another type of `UPDATE`/`DELETE` conflict is when a `DELETE` for a row comes after that row was updated locally. In this situation, the outcome depends on the type of conflict detection used and the replication identity in use. 
+
+When using [origin conflict detection](#origin-conflict-detection) (the default) and only primary keys for replication, no conflict is detected, leading to the `DELETE` being applied and the row being removed. 
+
+If `REPLICA IDENTITY FULL` is being used for replication, the conflict will be detected and a `delete_recently_updated` conflict is generated.
```

Comment:
This is not accurate

Reply:
The conflict is detected when REPLICA IDENTITY FULL is used and one of the following things are true:
- table uses row version conflict detection/resolution (this anyway requires REPLICA IDENTITY FULL)
- table uses per-column conflict detection/resolution aka CLCD (this anyway requires REPLICA IDENTITY FULL)
- table uses custom conflict resolution triggers